bzl/workspace.bzl

```diff
@@ -61,12 +61,14 @@ def plaidml_workspace():
         strip_prefix = "gflags-e292e0452fcfd5a8ae055b59052fc041cbab4abf",
     )
 
+    # TODO: When gmock is updated, see if its LICENSE file has been exported;
+    #       if it has, delete //bzl:googletest.LICENSE, and modify downstream dependents to
+    #       pull the license directly from the gmock release.
     http_archive(
         name = "gmock",
-        url = "https://github.com/google/googletest/archive/release-1.8.0.tar.gz",
-        sha256 = "58a6f4277ca2bc8565222b3bbd58a177609e9c488e8a72649359ba51450db7d8",
-        strip_prefix = "googletest-release-1.8.0",
-        build_file = clean_dep("//bzl:gmock.BUILD"),
+        url = "https://github.com/google/googletest/archive/release-1.10.0.tar.gz",
+        sha256 = "9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb",
+        strip_prefix = "googletest-release-1.10.0",
```

Comment:
Let's avoid making dependency changes until we're sure that OpenVINO tests are ready for the switch.

Reply:
Done

Reply:
OpenVINO tests will need 1.10 googletest when we advance our OV reference, I have a variation of this change on my OV branch.

That said it's possible that this change will break some OV tests until I get fixes in; I'm working on getting the OV advancement PR ready as my main priority right now. But if you aren't seeing CI issues from this, it's probably fine.